src/coreclr/System.Private.CoreLib/src/System/Reflection/TypeNameParser.CoreCLR.cs

```diff
@@ -80,6 +84,54 @@ namespace System.Reflection
             }.Parse();
         }
 
+        internal static RuntimeType GetTypeByNameUsingCARules(string typeName, RuntimeModule scope)
```

Comment:
I am assuming that "CA" means custom attribute.

Are the custom attribute rules defined somewhere? If not, is this method indicative of those rules?

Reply:
Yes, CA means custom attribute. I have fixed the name to be less cryptic.

The CA search rules are same as `Type.GetType` search rules. I have included the description of how this lookup differs from standard `Type.GetType` in the comment.